### PR TITLE
Add command line tool

### LINKED ISSUE #37: Create a command line tool for compiling Tornado templates.

### PR DESCRIPTION
**Pull request by smfoote**
This tool is based largely on dustc, but doesn't yet have as many
features. Fixes #37

### CODE REVIEW

File: bin/twister.es6 [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  file.on('end', function() {
+    cb(null, data);
+  });
+}
+
+function output(data, file) {
+  let outputStream;
+  try {
+    if (file) {
+      outputStream = streams[file] ? streams[file] : streams[file] = cli.native.fs.createWriteStream(file);
```
Comment (jimmyhchan) [inline bin/twister.es6]: too much fanciness on one line?

File: bin/twister.es6 [hunk 2]
```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env node
+
```
Comment (jimmyhchan) [inline bin/twister.es6]: IMO, there's an expectation this'll be named `bin/tornado` or `bin/tornado-cli`  or `bin/cli`

File: bin/twister [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+function glob(globPaths) {
+  return globPaths.map(function (arg) {
+    return cli.glob.sync(arg);
+  }).reduce(function (a, b) {
+    return a.concat(b);
+  }, []);
+}
+
+function compile(data, name) {
```
Comment (jimmyhchan) [inline bin/twister]: nit pick: can we move this down.  the other fns are all IO related and this one is tornado related.

File: bin/twister.es6 (hunk 1 above)
Comment (smfoote) [inline bin/twister.es6]: Fixed.

File: bin/twister.es6 (hunk 2 above)
Comment (smfoote) [inline bin/twister.es6]: Agreed, and fixed.

File: bin/twister (hunk 3 above)
Comment (smfoote) [inline bin/twister]: Done.

**Pull request merged**